src/com/yetanalytics/lrs_admin_ui/views/util/table.cljs

```diff
@@ -0,0 +1,42 @@
+(ns com.yetanalytics.lrs-admin-ui.views.util.table
+  (:require
+   [reagent.core :as r]
+   ["react-data-table-component" :as DataTable]))
+
+(def default-table-style
+  {:header    {:style {:minHeight "56px"}}
+   :headRow   {:style {:borderTopStyle "solid"
+                       :borderTopWidth "1px"
+                       :borderTopColor "lightgray"
+                       :fontWeight "bold"}}
+   :headCells {:style {"&:not(:last-of-type)" {:borderRightStyle "solid"
+                                               :borderRightWidth "1px"
+                                               :borderRightColor "lightgray"}}}
+   :cells     {:style {"&:not(:last-of-type)" {:borderRightStyle "solid"
+                                               :borderRightWidth "1px"
+                                               :borderRightColor "lightgray"}}}});
+
+(defn data-table
+  [{:keys [columns expandableRowsComponent data customStyles] :as opts
+    :or   {customStyles default-table-style}}]
```

Comment:
There should be a docstring here, especially since this _is_ a very complex and confusing component.

Reply:
8de3e861b0685ac4fdc5b17347fae154962ae780